spec.md

```diff
@@ -268,7 +268,13 @@ Each SP MUST provide:
 
 * **Node Plugin**: A gRPC endpoint serving CSI RPCs that MUST be run on the Node whereupon an SP-provisioned volume will be published.
 * **Controller Plugin**: A gRPC endpoint serving CSI RPCs that MAY be run anywhere.
-* In some circumstances a single gRPC endpoint MAY serve all CSI RPCs (see Figure 3 in [Architecture](#architecture)).
+
+Each SP MAY provide:
+
+- **GroupController Plugin**: A gRPC endpoint serving CSI RPCs that MAY be run anywhere.
```

Comment:
Hi @xing-yang,  Is the `GroupController plugin` optional or required for a SP?

Reply:
It is optional.

Reply:
thanks